pom.xml

```diff
@@ -39,6 +39,13 @@
             <version>2.5.4</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.springframework.retry</groupId>
+            <artifactId>spring-retry</artifactId>
+            <version>1.3.1</version>
```

Comment:
In to next PR could be useful to put all versions into up section `<properties>` of `pom.xml`

Reply:
ok